http-server/src/main/java/io/airlift/http/server/ReloadableSslContextFactoryProvider.java

```diff
@@ -80,8 +103,14 @@ public ReloadableSslContextFactoryProvider(HttpServerConfig config, ScheduledExe
     private void loadContextFactory(SslContextFactory.Server sslContextFactory)
     {
         KeyStore keyStore = loadKeyStore(keystoreFile.map(FileWatch::getFile), keystorePassword, keyManagerPassword);
+
+        String password = firstNonNullOrEmpty(keyManagerPassword, keystorePassword);
```

Comment:
This method name is a bit confusing. How about `firstNonNullOrElseEmpty`

Or maybe inline since it's only used in one place now
```java
String password = "";
if (keyManagerPassword != null) {
    password = keyManagerPassword;
}
else if (keystorePassword != null) {
    password = keystorePassword;
}